firstvoices/backend/views/search/base_search_views.py

```diff
@@ -354,6 +354,32 @@
                     ),
                 ],
             ),
+            OpenApiParameter(
+                name="hasUnrecognizedChars",
+                description="Filter dictionary entries that have a unrecognized character present in their title.",
```

Comment:
Minor typo: This should say "...have **an** unrecognized..." Same with the descriptions below.